### PR TITLE
Make '_events' 'readonly'.

### PR DESCRIPTION
**Pull request by kreghek**
https://app.codacy.com/manual/Tektronix/initialstate.NET/issues?&filters=W3siaWQiOiJMYW5ndWFnZSIsInZhbHVlcyI6WyJDU2hhcnAiXX0seyJpZCI6IkNhdGVnb3J5IiwidmFsdWVzIjpbIkVycm9yUHJvbmUiXX0seyJpZCI6IkxldmVsIiwidmFsdWVzIjpbbnVsbF19LHsiaWQiOiJQYXR0ZXJuIiwidmFsdWVzIjpbbnVsbF19LHsiaWQiOiJBdXRob3IiLCJ2YWx1ZXMiOltudWxsXX1d

### CODE REVIEW

**Comment by zkoppert** [comment]
@kreghek Thanks so much for contributing!! Can you go to this link and sign the CLA so that we can accept the changes? Thanks!

https://cla-assistant.io/tektronix/initialstate.NET?pullRequest=16

**Review by dwyban — CHANGES_REQUESTED** [review]
Please combine with #13, #14 and #15 changes.

File: InitialState.NET/ISEventDataCollection.cs [hunk 1]
```diff
@@ -28,7 +28,7 @@ namespace InitialState.Streaming
     [System.ComponentModel.ListBindable(false)]
     public class ISEventDataCollection : System.Collections.Generic.IList<ISEventData>
     {
-        private List<ISEventData> _events = null;
+        private readonly List<ISEventData> _events = null;
```
Comment (nlynchjo) [inline InitialState.NET/ISEventDataCollection.cs:31]:
```suggestion
        private readonly List<ISEventData> _events = 0;
```

Comment (dwyban) [reply InitialState.NET/ISEventDataCollection.cs:31]: Hey nlynchjo!  Why =0 and not = null?  I suppose the run-time will make it null automatically at creation so it probably shouldn't even be assigned really...

Comment (kreghek) [reply InitialState.NET/ISEventDataCollection.cs:31]: > 

`_events` is `List<T>`. It cant be assigned 0.

Comment (nlynchjo) [reply InitialState.NET/ISEventDataCollection.cs:31]: > 
> 
> Hey nlynchjo! Why =0 and not = null? I suppose the run-time will make it null automatically at creation so it probably shouldn't even be assigned really...

So sorry!! I was supposed to remove this suggestion, this PR was just at the top of the list, and I was demonstrating commit suggestions which is unique to GitHub. I was supposed to remove the suggestion after and I forgot!

**Pull request closed without merging**